forem/forem
# Removed ordering of co-authors by id

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents



- Closes #20152

## QA Instructions, Screenshots, Recordings

[Problem Video](https://www.loom.com/share/b42c868029c14e20b8539b429870f972)
[Solution Video](https://www.loom.com/share/8af22f3c49b8419ebeef95746cd9aec9?sid=1aec3038-7df0-4854-b679-fcfc7a04f8a8)

**How to test this**?
1. Create an organisation
2. Add 2-3 users with different names in that organisation
3. Now create a post under the organisation.
4. Visit dashboard, on right side see a drop-down with `Personal`, change it to your organisation.
5. Click on 3-dots (overflow-icon) for the post that you created which will open the dropdown
6. Now add these newly created users such that you add the user which you created in last as first followed by the other user.
7. Visit this post and just above the title see that the name of the co-authors is mentioned in the exact same order that you mentioned while adding them.


### UI accessibility checklist

No

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._
